### DISCUSSION TITLE
Instructions unclear for disk size in HomeAssistant script

### DISCUSSION THREAD

**Discussion by DeepCoreSystem** [tteck/Proxmox#168] [General]
It is unclear that the disk size should be given in GBytes but without typing 'Gb' ...

https://github.com/tteck/Proxmox/blob/efb555914ac01ef4ac5559fc77499b4cd4a17962/ct/homeassistant-v3.sh#L150

**Comment by tteck** [comment]
yeah, it's just the number. Same for cores (vCPU) and ram (MiB)

**Comment by tteck** [comment]
I made some changes to hopefully make it clearer what to enter
